Controls/DataControls/DataControls.UWP/ListView/View/Controls/RadListViewItem.IDragDropElement.cs

```diff
@@ -267,11 +267,36 @@ void IDragDropElement.OnDragDropComplete(DragCompleteContext context)
                 this.FinalizeReorder(context);
 
                 var isExecuted = false;
+                var destinationItem = default(RadListViewItem);
+                var placement = default(ItemReorderPlacement);
 
-                if (data.InitialSourceIndex != data.CurrentSourceReorderIndex)
+                if (data.InitialSourceIndex < data.CurrentSourceReorderIndex)
+                {
+                    destinationItem = this.GetDestinationItem(data.CurrentSourceReorderIndex - 1);
```

Comment:
As GetDestinationItem's implementation has changed and now only has one line of code I believe we can delete the method and add something like that:
```
var host = this.reorderCoordinator.Host;
if (data.InitialSourceIndex != data.CurrentSourceReorderIndex)
{
    destinationItem = (RadListViewItem)host.ElementAt(data.CurrentSourceReorderIndex - 1);
    ...
}
```